src/trace/descriptor.ts

```diff
@@ -36,7 +36,17 @@ export function augmentLogs(logs: StructLog[], constants: object): { logs: Log[]
     const nextStack = nextLog ? nextLog.stack : undefined;
 
     let { extended, info: nInfo } = describeOperation(log, nextStack || [], info);
-    const nLog: Log = new Log(log, extended, info.lastLog);
+    let nLog: Log = new Log(log, extended, info.lastLog);
+
+    if (nextLog != undefined) {
+      nLog.gasCost = nextLog.gasCost;
```

Comment:
Maybe we should keep both params and make this like "opGasCost" since it'll kind of differ from the original (totally odd) meaning of `gasCost` from the raw trace data?